src/main/java/htsjdk/samtools/seekablestream/SeekableStreamFactory.java

```diff
@@ -40,6 +43,14 @@
 public class SeekableStreamFactory{
 
     private static final ISeekableStreamFactory DEFAULT_FACTORY;
+    private static final String HTTP = "http";
+    private static final String HTTPS = "https";
+    private static final String FTP = "ftp";
```

Comment:
See below -- recommend including "://" in these constants to guard against files named after a protocol

Reply:
instead of adding them to the constants which are also used directly compared against getScheme the isHandledBy method now appends '://' to them.